sourmash/lca/lca_db.py

```diff
@@ -291,6 +309,7 @@ def save(self, db_name):
             save_d['idx_to_lid'] = self.idx_to_lid
             save_d['lid_to_lineage'] = self.lid_to_lineage
             
+            print("\n\nPYTHON:\n", save_d, "\n\n")
```

Comment:
If you want to compare the JSON generated by both, it might be better to use `json.dumps` here, print the generated buffer, and then write the buffer to `fp`. At this point `save_d` is still a Python dict, so while it is comparable it is not exactly JSON yet.